Edit animation intro page for clarity and grammar (#4472)
godotengine/godot-docs: tutorials/animation/introduction.rst

-  The timeline and track controls, where you can zoom the timeline and
   edit tracks, for example.

Computer animation relies on keyframes
--------------------------------------

A keyframe defines the value of a property at a point in time.

Diamond shapes represent keyframes in the timeline. A line between two
keyframes indicates that the value doesn't change between them.

.. figure:: img/animation_keyframes.png
   :alt: Keyframes in Godot

   Keyframes in Godot

You set values of a node's properties and create animation keyframes for them. 
When the animation runs, the engine will interpolate the values between the 
keyframes, resulting in them gradually changing over time.

.. figure:: img/animation_illustration.png
   :alt: Two keyframes are all it takes to obtain a smooth motion

   Two keyframes are all it takes to obtain a smooth motion

The timeline defines how long the animation will take. You can insert keyframes 
at various points, and change their timing. 

.. figure:: img/animation_timeline.png
   :alt: The timeline in the animation panel

   The timeline in the animation panel

Each line in the Animation Panel is an animation track that references a 

> Normal or Transform property of a node. The track's name or id is a path to 
> the node and its affected property. For example, the position track 
> in the illustration refers to to the Position Transform of the Sprite 
> node.

Reviewer: ```suggestion
Normal or Transform property of a node. Each track stores a path to 
a node and its affected property. For example, the position track 
in the illustration refers to to the ``position`` property of the Sprite 
node.
```

.. tip::

>    If you animate the wrong property, you can edit a track's path at any time;
>    just double click on it and type the new path. Play the animation using the

Reviewer: ```suggestion
   If you animate the wrong property, you can edit a track's path at any time
   by double-clicking on it and typing the new path. Play the animation using the
```

Scene setup
~~~~~~~~~~~

For this tutorial, we'll create an AnimationPlayer node with a sprite node as 
its child. We will animate the sprite to move between two points on the screen. 

.. figure:: img/animation_animation_player_tree.png
   :alt: Our scene setup

   Our scene setup

.. tip::

   Animated nodes don't have to be children of the AnimationPlayer node, 
   but putting them there is a nice way of distinguishing animated nodes from
   non-animated nodes in the Scene Tree.

The sprite holds an image texture. For this tutorial, select the Sprite node,
click Texture in the Inspector, and then click Load. Select the default Godot
icon for the sprite's texture. 

Select the AnimationPlayer node and click the "Animation" button in the

> animation editor. From the list select "New" (|Add Animation|) to add a new 
> animation. Enter a name for the animation in the dialog box.

Reviewer: ```suggestion
animation editor. From the list, select "New" (|Add Animation|) to add a new 
animation. Enter a name for the animation in the dialog box.
```

   Convenience buttons

These switches and buttons allow you to add keyframes for the selected

> node's location, rotation, and scale. Since we are only animating the sprite's 
> position, make sure that only the location switch is selected. (The selected 
> switches are blue.)

Reviewer: ```suggestion
node's location, rotation, and scale. Since we are only animating the sprite's 
position, make sure that only the location switch is selected. The selected 
switches are blue.
```

Click on the key button to create the first keyframe. Since we don't have a 

> track set up for the Transform/Position property yet, Godot will offer to 
> create it for us. Click **Create**. 

Reviewer: ```suggestion
track set up for the Position property yet, Godot will offer to 
create it for us. Click **Create**. 
```

The second keyframe
~~~~~~~~~~~~~~~~~~~

> Now we need to set the our sprite's end location and how long it will take for 
> it to get there.

Reviewer: ```suggestion
We need to set our sprite's end location and how long it will take for it to get there.
```

> Let's say we want it to take two seconds to move between the points. By
> default the animation is set to last only one second, so change the animation 
> length to 2 in the controls on the right side of the animation panel's timeline 
> header.

Reviewer: ```suggestion
Let's say we want it to take two seconds to move between the points. By
default, the animation is set to last only one second, so change the animation 
length to 2 in the controls on the right side of the animation panel's timeline 
header.
```

.. figure:: img/animation_set_length.png
   :alt: Animation length

   Animation length

> Now move the sprite right, to its final position. You can use the Move tool in the 
> editor or set the Transform/Position/X value in the Inspector. 

Reviewer: ```suggestion
Now, move the sprite right, to its final position. You can use the *Move tool* in the 
toolbar or set the *Position*'s X value in the *Inspector*.
```

Keyframes for other properties
------------------------------

> Godot animation isn't restricted to transformation properties. You can create a 
> track and set keyframes to animate any property.

Reviewer: ```suggestion
Godot's animation system isn't restricted to position, rotation, and scale.
You can animate any property.
```

> If you select your sprite while the animation panel is visible, Godot will 
> display a small keyframe button in the Inspector for each the sprite's 
> properties. Click on one of these buttons and Godot will automatically add a 
> track and keyframe to the current animation.

Reviewer: ```suggestion
If you select your sprite while the animation panel is visible, Godot will 
display a small keyframe button in the *Inspector* for each of the sprite's 
properties. Click on one of these buttons to add a track and keyframe to 
the current animation.
```

Edit keyframes
--------------

> For advanced use you can click on a keyframe in the time line to display
> the keyframe editor in the Inspector. You can us this editor to directly edit
> the keyframe's values.

Reviewer: ```suggestion
You can click on a keyframe in the animation timeline to display and
edit its value in the *Inspector*.
```

.. figure:: img/animation_keyframe_editor_key.png
   :alt: Keyframe editor editing a key

   Keyframe editor editing a key

> You can also edit the easing value for a keyframe here by clicking and dragging 
> its easing setting. This tells Godot how to change the property values when it 
> reaches this keyframe.

Reviewer: ```suggestion
You can also edit the easing value for a keyframe here by clicking and dragging 
its easing curve. This tells Godot how to interpolate the animated property when it 
reaches this keyframe.
```
